app/models/mail_template.rb

```diff
@@ -0,0 +1,31 @@
+class MailTemplate < ActiveRecord::Base
+  # t.string   :name
+  # t.string   :template_type
+  # t.string   :from
+  # t.string   :to
+  # t.string   :subject
+  # t.string   :attachment
+  # t.text     :text
+  # t.integer  :organization_id
+  # t.timestamps
+
+  TEMPLATE_TYPES = ['invoice', 'reminder']
+
+  attr_accessible :name, :template_type, :from, :to, :subject, :attachment, :text
+
+  belongs_to :organization
+
+  validates :name, presence: true, uniqueness: { scope: :organization_id}
+  validates :template_type, inclusion: { in: TEMPLATE_TYPES }
+  validates :from, presence: true
+  validates :to, presence: true
```

Comment:
but `from` will always be the `organization` email right?
and `to` will always be the the receiver of the invoice.

I haven't tried the GUI, but maybe we don't need to have database columns for them?

`to` and `from` could just be `methods` on the model maybe?

```ruby
def to
  'sale.contact_email'
end

def from
  organization.email
end
```

then we wouldn't have to do any validations on them.